Makefile

```diff
@@ -34,13 +34,17 @@ images:
 PROTO_DEFS := $(shell find . $(DONT_FIND) -type f -name '*.proto' -print)
 PROTO_GOS := $(patsubst %.proto,%.pb.go,$(PROTO_DEFS))
 
+# Ensure a each image that requires a migration directory mounted has the updated migrations directory in their
```

Comment:
It took me a while to work out what was going on.  Maybe point to the `cp -r` line lower down which does the actual work?